pagopa/io-app
# chore: [IOPID-2050] Add session refresh Feature Flag

> [!Warning] 
> This PR depends on https://github.com/pagopa/io-services-metadata/pull/815
> When https://github.com/pagopa/io-services-metadata/pull/815 will be released i need to change the version into the "content_specs" in package.json 


## List of changes proposed in this pull request
- Edit package.json with correct version of io-services-metadata
- Add selector to remote FF (`fastLoginSessionRefreshFFEnabled`)
- Add selector to control if fast login is active (`isFastLoginSessionRefreshEnabledSelector`)
- Add toggle to enabled/disabled this feature (this toggle now is visible only if the remote FF is active because otherwise it would not display the feature, so it will be displayed in the next pr (is controlled using`isFastLoginSessionRefreshToggleActiveSelector` -> is a persisted value)

## How to test
- Run the application using the latest version of the io-dev-api-server
- delete [this line](https://github.com/pagopa/io-app/blob/128dfab2bfdcc7bc7ba1783daf7eb05ccfa18b23/ts/screens/profile/DeveloperModeSection.tsx#L368) to see the toggle 
- follow the video 

## Demo
<video src="https://github.com/user-attachments/assets/d937f98f-6e0c-4ea9-b285-42c8b0d8b2a2"/>
